doc/da65.sgml

```diff
@@ -299,9 +312,10 @@ anything). Each attribute is terminated by a semicolon.
 
 <sect1>Comments<p>
 
-Comments start with a hash mark (<tt/#/); and, extend from the position of
-the mark to the end of the current line. Hash marks inside of strings will
-<em/not/ start a comment, of course.
+Comments start with a hash mark (<tt/#/) or a double slashe (<tt>//</tt>);
```

Comment:
I'm no native speaker but I think it needs to read "double slash".

Reply:
Fixed.